src/QsCompiler/Tests.LanguageServer/ProjectLoaderTests.cs

```diff
@@ -316,7 +315,6 @@ public async Task LoadQSharpConsoleAppAsync()
                 Path.Combine(projDir, "Operation11.qs"),
             };
 
-            Assert.IsTrue(context.UsesIntrinsics());
```

Comment:
Why did these asserts need to be removed?

Reply:
Because the updated version does not use some `*Intrinsic*.dll` (I don't remember the exact name, hence the wildcard). And that DLL does not exist any more in the later packages.

Reply:
Whether the DLL is present or not doesn't seem relevant here. Are you saying this method on the context no longer exists, and intrinsics are always used by default now?

Reply:
The `UsesIntrinsics()` is defined like [this](https://github.com/microsoft/qsharp-compiler/blob/fa13bf673983247c5f3b570eb78cb0e692e9b859/src/QsCompiler/Tests.LanguageServer/ProjectLoaderTests.cs#L418). 

During debugging I got an impression that this check verifies if the test project still has either the "Microsoft.Quantum.Intrinsic.dll" or "Microsoft.Quantum.Primitives.dll" as a dependency (I believe those are the parts of the very old QDK). After an update of test11 to the new version of QDK (that has no those DLLs any more) the check has started failing. So I decided to remove that check.

If we leave the test11 as is (revert in this PR the test11 change and the change being discussed) then I believe one particular Component Governance issue will continue being reported (the one that you can navigate to from the DevOps item 48477, then search for "test11" in the Component Governance issue).